huggingface/script/run_mlm.py

```diff
@@ -224,10 +228,6 @@ def main():
     else:
         model_args, data_args, training_args = parser.parse_args_into_dataclasses()
 
-    # Sending telemetry. Tracking the example usage helps us better allocate resources to maintain them. The
-    # information sent is the one passed as arguments along with your Python/PyTorch versions.
-    send_example_telemetry("run_mlm", model_args, data_args)
```

Comment:
Is it preferred to add the telemetry back in?

Ah, it's kind of a side-effect of my version of "cleaning" the branch of unnecessary commits. the telemetry updates were added after I made my original changes to the script, and I wanted to recreate the process of copying over the original scripts then updating them with the Optimum updates, so that it would be easier for reviewers to see the Optimum support changes. But it is trivial for me to add the telemetry back in.

Reply:
Yeah, please add telemetry back if there is no conflict with your current work. Ideally we better not change anything not related to our pull request